tests/integration/courses.test.js

```diff
@@ -3905,6 +3927,58 @@ describe('COURSES ROUTES - PUT /courses/{_id}/companies', () => {
 
       expect(response.statusCode).toBe(400);
     });
+
+    it('should return a 404 if company is not in holding (intra_holding course)', async () => {
+      const response = await app.inject({
+        method: 'PUT',
+        url: `/courses/${coursesList[22]._id}/companies`,
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { company: authCompany._id },
+      });
+
+      expect(response.statusCode).toBe(404);
+    });
+  });
+
+  describe('HOLDING_ADMIN', () => {
+    it('should return a 200 if holding admin try to add company to intra_holding course', async () => {
```

Comment:
pourquoi `try to` ? s'il y parvient tu peux mettre simplement "should return a 200 if holding admin add company to intra_holding course"

Reply:
done